local-volume/helm/examples/gce-cleanbyjobs.yaml

```diff
@@ -0,0 +1,15 @@
+common:
+  useJobForCleaning: true
+classes:
+- name: local-scsi
+  hostDir: "/mnt/disks/by-uuid/google-local-ssds-scsi-fs"
+  blockCleanerCommand:
```

Comment:
On gce/gke, we actually separate block and preformatted volumes into different directories.  So this block cleaning config doesn't do anything.  It may be better to put this in a baremetal example.

Reply:
Should I remove all `blockCleanerCommand` sections from other gce/gke example yamls? I never run kubernetes on gce/gke, could you help test these examples if possible?

Reply:
Yeah I would leave it out for now and add it to the baremetal example instead.  I have an issue open on me to integrate and test the block feature on gce/gke

Reply:
Done in https://github.com/kubernetes-incubator/external-storage/pull/789/commits/427e9675173c9d7c734e8d2a7ca620ec077ce96d